dev/idm-am-common-user/development/amster-values.yaml

```diff
@@ -0,0 +1,9 @@
+  domain: .forgeops.com
+  configStore:
+    suffix:  "ou=identities"
```

Comment:
This was a mistake in my config.  It screwed up the placement of am's configuration. See this PR: https://github.com/ForgeRock/forgeops-init/pull/146